recipes/hda/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "hda" %}
+{% set version = "1.15" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 68911208696fed8461f76d53a1f28bf3af17e07aa8c8210acbbcf1145a4ca9b3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Since this is pure Python, it can be a noarch build:
```suggestion
  number: 0
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```